modules/kafka/src/main/resources/application.conf

```
@@ -0,0 +1,71 @@
+{
+  "input": {
+    "type": "Kafka"
+    "topicName": "collector-payloads"
+    "bootstrapServers": "localhost:9092"
+    "consumerConf": {
+      "enable.auto.commit": "false"
```

```suggestion
      "enable.auto.commit": "false"
      "group.id": "enrich"
```
Group id is [required](https://docs.confluent.io/platform/current/installation/configuration/consumer-configs.html#consumerconfigs_group.id) given that we use `.subscribe()`.
Also, it would be good to make it appear in the extended example so that users can see what the default value is.